### ISSUE TITLE
Docs website issues for colourblind users

### ISSUE THREAD

**Issue by madscientistproductions** [jfbrennan/m-#39]
For colourblind people (especially those with Protanopia), the current Ace editor theme (ace-monokai) used in the Documentation has  #F92672 on black which suffers from a low contrast ratio and is very difficult to read (even when highlighting).

I'd suggest something like solarized dark or merbivore, or if you want to get customised, ayu-theme: https://github.com/ayu-theme/ayu-ace

Obviously this is just my opinion, so you're free to discard at will, however I will say that's it's currently quite difficult to read the demo code. :)

**Comment by jfbrennan** [comment]
I hear ya, my son is colorblind too. Monokai gets used a lot I think because HTML looks so much better compared to others. Is there a shade of black background that #F92672 becomes legible for you? Or maybe something close to #F92672 that works with the current background?

**Comment by madscientistproductions** [comment]
Hey there, thanks for the reply!  I think something like #fe9fc1 (with a contrast ratio of 7.75) could work better for this. 

.ace-monokai .ace_entity.ace_name.ace_tag, .ace-monokai .ace_keyword, .ace-monokai .ace_meta.ace_tag, .ace-monokai .ace_storage {
    color: #FE9FC1;
}

It's still a shade of red, but veering towards pink.  I couldn't find a good background shade that works unfortunately, although I'm not a graphics designer, so.. :)

Obviously this is only my opinion, so I'd get a few others to check out the colour scheme first :)

**Comment by jfbrennan** [comment]
Can't seem to get any color theme to look good for both colorblind and non-colorblind users I'm closing
